src/Compilers/Core/Portable/Operations/IVariableDeclarationGroup.cs

```diff
@@ -7,16 +7,20 @@ namespace Microsoft.CodeAnalysis.Semantics
     /// <summary>
     /// Represents a local variable declaration statement.
     /// </summary>
+    /// <para>
+    /// Current Usage:
+    ///   (1) C# Local declaration statement
+    ///   (2) VB Dim statement
+    /// </para>
```

Comment:
></para> [](start = 8, length = 7)

Do we also use it in fixed and using? Anything else?

Reply:
Doesn't appear so in our test cases. Added those 2.

---
In reply to: [146932266](https://github.com/dotnet/roslyn/pull/22842#discussion_r146932266) [](ancestors = 146932266)